doc/en/api/1.0.0/owsservices.yaml

```diff
@@ -980,6 +980,233 @@ paths:
         200:
           description: OK
 
+  /services/wmts/settings:
```

Comment:
Q: Long term should this file be broken up into multiple individual files?

Reply:
Longer term we should provide OpenAPI 3 yaml files, and use tags to split the api into sub-apis, so it wouldn't matter on which files each path is. But would still probably keep these ones on the same file